### PR TITLE
Dos2unix 7.3

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/dos2unix.rb [hunk 1]
```diff
@@ -13,8 +13,8 @@ class Dos2unix < Formula
   end
 
   devel do
-    url "http://waterlan.home.xs4all.nl/dos2unix/dos2unix-7.3-beta3.tar.gz"
-    sha256 "4b34d0aa81891795566982af901a84efd54344e1e1bcba138329eb5afe0fdc68"
+    url "http://waterlan.home.xs4all.nl/dos2unix/dos2unix-7.3.1-beta5.tar.gz"
```
Comment (DomT4) [inline Library/Formula/dos2unix.rb]: Does this need a `version` tag? Stuff appended with `-beta` can be problematic for the detection.

Comment (dunn) [inline Library/Formula/dos2unix.rb]: Looks like it scanned OK: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/32900/version=mountain_lion/testReport/brew-test-bot/mountain_lion/install___devel_dos2unix/

Comment (DomT4) [inline Library/Formula/dos2unix.rb]: Nice! Feel free to :ship: this if you want to, otherwise I'll grab it in a bit.

**Comment by dunn** [comment]
Merged in 22642a910241ee7d9130456cab618d5cd720abfd. Thanks again 🎈

File: Library/Formula/dos2unix.rb (hunk 1 above)
Comment (nijikon) [inline Library/Formula/dos2unix.rb]: @DomT4 It worked yes, I've checked, but thanks for keeping your eyes open :)

**Pull request closed without merging**